etl-job/fhir_import_export.py

```diff
@@ -270,6 +288,7 @@ def _load_all(study: str,
     return True
 
 
+# TODO: do we want to delete this as well? (TKEDTE-316)
```

Comment:
should this `_get` function be deleted as well? I don't see any reference to it in g3t but I could be mistaken.

Reply:
Looking through the g3t codebase I don't see where it is being used currently, so yes it can probably be removed. Before this PR is merged double check it with Walsh.

Reply:
Please double check that the `g3t clone`  works.
https://github.com/ACED-IDP/gen3_util/blob/451196afd8f4a9a0aa4193cdc9a219f9e44fb1c3/gen3_tracker/git/cloner.py#L42